src/tcgwars/logic/impl/gen4/CallOfLegends.groovy

```diff
@@ -1584,7 +768,18 @@ public enum CallOfLegends implements LogicCardInfo {
             energyCost C
             attackRequirement {}
             onAttack {
-              damage 0
+              flip {
+                delayed {
+                  before PLAY_TRAINER, {
+                    if (bg.currentTurn == self.owner.opposite) {
+                      wcu "Fake Tears prevents you from playing Trainer cards."
+                      prevent()
+                    }
+                  }
+                  unregisterAfter 2
+                }
```

Comment:
"Trainer Cards" in these formats actually means "ITEM" cards I'm pretty sure. Supporter cards and Stadium cards were not considered Trainers at the time and did not have TRAINER printed on their cards.